Python/errors.c

```diff
@@ -1444,12 +1444,15 @@ make_unraisable_hook_args(PyThreadState *tstate, PyObject *exc_type,
 
    It can be called to log the exception of a custom sys.unraisablehook.
 
-   Do nothing if sys.stderr attribute doesn't exist or is set to None. */
+   This assumes 'file' neither NULL nor None.
```

Comment:
```suggestion
   This assumes 'file' is neither NULL nor None.
```

Sorry, I forgot the "is"....